.storybook/index.scss

```diff
@@ -48,3 +50,13 @@
     }
   }
 }
+
+/*** TEMPORARY ***/
+
+.bx--css-grid {
```

Comment:
Question - is this a relic from local experimentation or is it required as an override or workaround?

Reply:
These are styles taken from `bx--grid` that don't seem to be included for `bx--css-grid`. I guess I marked Temporary just cause I haven't looked into other ways to bring them in

Reply:
Ah, I see! Thanks for clarifying - could you include `bx--grid` as an additional class on the grid container?

Reply:
Oh, I suppose I can, but is it not the intention to try to stay away from the old `bx--grid` implementation? Is it that it's okay to keep `bx--grid` around for its styles since it doesn't actually utilize flex?

Reply:
You're right, that's definitely the intention so we'd like to avoid it where possible - which styles are required, just the `max-width`?

I guess this hangs on Carbon's intentions for grid `max-width`s moving forward. @tay1orjones could you provide some insight here?

Reply:
@SimonFinney Sorry for the wildly late reply on this, unfortunately got lost in my notifications 😓 

Elements with `bx--css-grid` shouldn't be given the `bx--grid` class. The implementations are wholly separate from one another. 

`max-width` was missed and so wasn't included in the initial drop but has since been added back in as of https://github.com/carbon-design-system/carbon/pull/8797. There's been quite a few improvements and fixes since then too.